bin/edit.js

```diff
@@ -15,17 +15,31 @@ module.exports = Cli.createCommand('edit', {
             required: false
         },
     },
+    optionGroups: {
+        'Output options': {
+            'version': {
```

Comment:
I don't think we should call this flag `--version` / `-v` to leave room for an eventual solution for webtask versioning. I think a better option would be `--editor-version` and since this is pretty unusual, no alias would be needed.

As for description, how about:

> Open the webtask in a specific version of the Webtask Editor